docs/passphrases.rst

```diff
@@ -11,7 +11,9 @@ each role in a SecureDrop installation.
           password manager included in Tails, to generate and retain
           strong and unique passphrases. We have created a template
           passphrase database that you can use to get started. For more
-          information, see the :doc:`tails_guide`.
+          information, see the :ref:`KeePassX setup instructions <keepassx_setup>`,
+          which are identical for the *Admin Workstation* and the
+          *Journalist Workstation*.
```

Comment:
We don't want the admin to set up the keepass database once and then copy it to the journalist drives - this will leave the journalists with many admin credentials - can we add a note/warning to this effect?

Reply:
I've tried to clarify the process in the journalist onboarding doc, where I think admins are most likely to see it (it is the primary section of the docs that describes the process of creating new *Journalist Workstations*). See 5cb607c9a6e6ef07c67b0937f22ddc40497e3614 - how does that look to you?